schemas/user.js

```diff
@@ -0,0 +1,25 @@
+const mongoose = require("mongoose");
+
+const userSchema = mongoose.Schema({
+  _id: mongoose.Schema.Types.ObjectId,
+  firstName: String,
```

Comment:
The idea of this users task was to move strictly defined user model from previous homework to mongoose.  Then add "favoriteProducts", "viewedProducts" and "orders" only. Not to add a bunch of new unnecessary fields.